kernel/kernel-impl/src/main/java/org/sakaiproject/authz/impl/BaseAuthzGroup.java

```diff
@@ -1046,6 +1048,16 @@ public void addMember(String user, String roleId, boolean active, boolean provid
 			grant.active = active;
 			grant.provided = provided;
 		}
+		
+
+		//send message to (ignite) MicrosoftMessagingService
+		this.baseAuthzGroupService.microsoftMessagingService().send(MicrosoftMessage.ADD_MEMBER_TO_AUTHZGROUP, MicrosoftMessage.builder()
```

Comment:
Shall we try/catch this or is it safe?

Reply:
It's safe. Try-catch is done inside MicrosoftMessagingService.send